packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
+declare var jestProjectConfig: Config.ProjectConfig;
 
 import path from 'path';
 import repl from 'repl';
 import vm from 'vm';
+import {Transformer} from '@jest/transform';
+import {Config} from '@jest/types';
 
-let transformer;
+let transformer: Transformer;
 
-const evalCommand = (cmd, context, filename, callback, config) => {
+const evalCommand = (
+  cmd: string,
+  _context: any,
+  _filename: string,
+  callback: (e: Error | null, result?: any) => void,
+  _config: Config.GlobalConfig,
+) => {
   let result;
   try {
     if (transformer) {
       cmd = transformer.process(
         cmd,
         jestGlobalConfig.replname || 'jest.js',
         jestProjectConfig,
-      );
+      ) as string;
```

Comment:
Why cast?

Reply:
either a bug in `@jest/transform`'s types, or this module will potentially crash

Reply:
This function call returns type `string | TransformedSource`. I don't have `TransformedSource` available to me.

Reply:
do `thing typeof 'string' ? thing : ting.code` or something similar

Reply:
oh that won't work in this case. I've typed `cmd` as a string here so that isn't compatible with `string | TransformedSource`. Unless I have `TransformedSource` type, I wont be able to type `cmd` to the same thing.

Reply:
pushed a fix for it 🙂